scripts/check-setup.sh

```
@@ -0,0 +1,441 @@
+        done
+    else
+        record_result "FAIL" "dotfilesディレクトリが見つかりません - インストールスクリプトを実行してください"
+    fi
+}
+
+# パフォーマンスの確認
+check_performance() {
+    log_step "システムパフォーマンスを確認中..."
+    
+    # メモリ使用量
+    local mem_used=$(free | grep Mem | awk '{printf "%.1f", $3/$2 * 100.0}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)